learn/advanced/datatypes.md

```diff
@@ -95,7 +95,7 @@ Will be processed as if all elements were arranged at the same level:
 
 Once the above array has been flattened, it will be parsed exactly as explained in the [string example](/learn/advanced/datatypes.md#examples).
 
-### Objects
+## Nested objects
```

Comment:
I think we should keep the name as is so it's consistent w/ the other datatypes

```suggestion
## Objects
```

Reply:
To make this easier to find by searching the term "nested fields", maybe we can change line 116:

```
During indexation, Meilisearch uses dot notation to eliminate nested fields:
```